R/load_data.R

```diff
@@ -0,0 +1,16 @@
+#'@export
+#'
+
+load_data <- function(){
+  
+  path <- rappdirs::user_data_dir("GeyseRdata.tsv","GeyseRdata")
+  
+  if(!file.exists(path)){
```

Comment:
While this check ensures that a file is available, it does not offer to either automatically download a more recent version and then overwrite the current file or at least notify the user of the function that a more recent version is available. I believe that's a must for efficient management of the database file 😃.

Reply:
I went ahead and implemented the changes you suggested.